src/transforms/kubernetes/mod.rs

```diff
@@ -1 +1,585 @@
 pub mod watch_client;
+
+use self::watch_client::{ClientConfig, PodEvent, RuntimeError, Version, WatchClient};
+use super::Transform;
+use crate::{
+    event::{self, Event, Value},
+    sources::kubernetes::POD_UID,
+    topology::config::{DataType, TransformConfig, TransformContext, TransformDescription},
+};
+use bytes::Bytes;
+use evmap::{ReadHandle, WriteHandle};
+use futures::{
+    compat::{Future01CompatExt, Stream01CompatExt},
+    future::{select, Either},
+    stream::StreamExt,
+};
+use k8s_openapi::api::core::v1::Pod;
+use rand::Rng;
+use serde::{Deserialize, Serialize};
+use snafu::{ResultExt, Snafu};
+use std::collections::{BTreeMap, VecDeque};
+use std::time::{Duration, Instant};
+use string_cache::DefaultAtom as Atom;
+use tokio::timer::Delay;
+
+// *********************** Defined by Vector **************************** //
```

Comment:
What's this all about?

Reply:
A leftover from times when configuration defined by Kubernetes was here. Now it's [here](https://github.com/timberio/vector/blob/6111a9c96a4460c0391e75be1d7b8d8f21b9d4ab/src/transforms/kubernetes/watch_client.rs#L18).